### ISSUE TITLE
[E.showMessage/showPrompt] Long message pushes button off screen

### ISSUE THREAD

**Issue by sir-indy** [espruino/BangleApps#1767]
### Affected hardware version

Bangle 2

### Your firmware version

2v13

### The bug

The sched library gives the option to add a message to an alarm/timer. Setting the message to a very long string overflows the screen, pushing the action buttons off the screen, making it impossible to stop or snooze the alert.

This message should be checked for length, then clipped or made scrollable.

This is the code I used to set the timer:
```
require("sched").setAlarm(timerID, {
    msg : "Spam, spam, spam, spam, eggs, spam, spam, spam, spam, spam, beans, spam, spam, spam, spam, spam, spam, spam, spam, spam.",
    vibrate : ".-.-",
    hidden: true,
    timer : seconds * 1000
  });
```

![download (1)](https://user-images.githubusercontent.com/53864146/165731442-fe53e308-69e8-47c8-b952-3ec87c12da48.png)
![download (2)](https://user-images.githubusercontent.com/53864146/165731454-baf95392-f470-461e-a14a-55259cd8b723.png)

**Comment by gfwilliams** [comment]
Thanks - I guess this could actually be considered an issue with `E.showMessage/showPrompt`?

But I guess this is reasonably low priority as it's a pretty contrived case (right now you have to input messages that long with code?)

**Comment by sir-indy** [comment]
You're right, I've renamed it to point at E.showMessage/showPrompt instead of sched.

You're also right that it's low priority, I did set the message in code, just wanted to flag it up.

**Comment by gfwilliams** [comment]
New showPrompt fixes this - had to add even more spam to trigger it but:

```
E.showPrompt("Spam, spam, spam, spam, eggs, spam, spam, spam, spam, spam, beans, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam, spam., spam, spam, spam, spam, spam, spam.");
```

<img width="176" height="176" alt="Image" src="https://github.com/user-attachments/assets/235ebc0c-324c-423f-b4b9-f0ad629ee0bc" />
